cmd/virt-launcher/virt-launcher.go

```diff
@@ -226,7 +229,16 @@ func main() {
 	domainConn := createLibvirtConnection()
 	defer domainConn.Close()
 
-	domainManager, err := virtwrap.NewLibvirtDomainManager(domainConn)
+	useEmulation := false
+	if _, err := os.Stat("/dev/kvm"); os.IsNotExist(err) {
+		if *allowEmulation {
+			useEmulation = true
+		} else {
+			glog.Fatalf("/dev/kvm is not present")
```

Comment:
"… and emulation not allowed"

Reply:
This log apply only when emulation is not allowed. If the /dev/kvm does not exist it is skipped.